src/style.css

```diff
@@ -0,0 +1,48 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  width: 100px;
+  height: 15px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.stars > * {
+  width: 13px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
```

Comment:
It is better not to use `*` selector for anything other than setting border-box property.
You have `stars_star` class. Use it here.